### DISCUSSION TITLE
Monte Carlo documentation

### DISCUSSION THREAD

**Discussion by malcolm-dsider** [NatLabRockies/GEOPHIRES-X#210] [Ideas]
A student requested that we include a Run Guide in the ReadMe file for the Monte Carlo simulations, similar to the existing GEOPHIRES guidelines

**Comment by softwareengineerprogrammer** [comment]
Do note that there is a Monte Carlo guide referenced from the README: https://github.com/NREL/GEOPHIRES-X?tab=readme-ov-file#monte-carlo. It can of course, always be improved, but perhaps the student in question could be made aware of that link.

**Comment by softwareengineerprogrammer** [comment]
Also note tracking issue https://github.com/NREL/GEOPHIRES-X/issues/72
